src/index.scss

```diff
@@ -0,0 +1,231 @@
+@import './variables';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: $font-family-roboto;
+}
+
+header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $white;
+  padding: 0 50px;
+  height: 60px;
+
+  .header__logo {
+    img {
+      height: 40px;
+    }
+  }
+
+  .nav {
+    &__list {
+      list-style: none;
+      display: flex;
+      padding: 0;
+      margin: 0;
+    }
+
+    &__item {
+      position: relative;
+      margin-left: $spacing-medium;
+    }
+
+    &__link {
+      text-decoration: none;
+      color: $text-color-dark;
+      text-transform: uppercase;
+      font-weight: 500;
+      padding: 10px 0;
```

Comment:
You're not defining the `font-size` for your `nav__link`.

Reply:
Also, make sure it has the `height: 100%;`, so the border can be in the right place. Also, the border is not matching the size of the mockup.